src/http/routes/route.ts

```diff
@@ -2,9 +2,11 @@ import { FastifyInstance } from 'fastify'
 import { authenticate } from '../controllers/authenticate-controller'
 import { registerUser } from '../controllers/register-user-controller'
 import { RegisterIndicatorController } from '../controllers/register-indicator-controller'
+import { RegisterLeadPublicController } from '../controllers/register-lead-public-controller'
 
 export async function appRoute(app: FastifyInstance) {
   app.post('/users', registerUser)
   app.post('/sessions', authenticate)
   app.post('/create/indicator', RegisterIndicatorController)
+  app.post('/create/lead/public', RegisterLeadPublicController)
```

Comment:
acho q n precisa do public no final, pode ficar como o do indicador
```suggestion
  app.post('/create/lead', RegisterLeadPublicController)
```